components/Contact.css

```diff
@@ -0,0 +1,13 @@
+.Section--Contact {
```

Comment:
-- is meant to signify a styling modifier on a Block (example: .Button--red). Contact is a name that would be more appropriately applied to an id. When thinking of classnames, try to think of the difference in appearance, and not the exact context in which it's used.